.buildkite/pipeline.yml

```diff
@@ -1,43 +1,36 @@
 steps:
   - name: ":test_tube: Test"
     key: test
+    command: go test -v -race .
```

Comment:
Turns out this step has been testing nothing! `go test -v -race ./...`

Reply:
Wow, it is a miracle they still pass.